libs/api/domains/housing-benefits/src/lib/models/housingBenefit.model.ts

```diff
@@ -0,0 +1,89 @@
+import { Field, GraphQLISODateTime, Int, ObjectType } from '@nestjs/graphql'
+import { CacheField } from '@island.is/nest/graphql'
+
+@ObjectType()
+export class HousingBenefitPageInfo {
+  @Field({ nullable: true })
+  hasNextPage!: boolean
+
+  @Field({ nullable: true })
+  hasPreviousPage?: boolean
+}
+
+@ObjectType()
+export class HousingBenefitPayment {
```

Comment:
All fields optional 🤔

Reply:
That's how it is generated from the service 😞 
I'll ask them if they can change it 🤝